src/main/java/org/jetbrains/research/anticopypaster/models/TensorflowModel.java

```diff
@@ -49,7 +58,8 @@ public float predict(FeaturesVector featuresVector) {
         Tensor<Float> x = Tensor.create(shape, floatBuffer);
 
         return runModel(session, x,
-                "serving_default_input_1:0", "StatefulPartitionedCall:0");
+                "serving_default_batch_normalization_input:0",
```

Comment:
it would be better to store it in a final field and give it a meaningful name (for each of the two strings)